File: code/src/test/scala/cern/colt/matrix/tlong/impl/SparseCCMLongMatrix2DViewTest.scala
package cern.colt.matrix.tlong.impl

//remove if not needed
import scala.collection.JavaConversions._

class SparseCCMLongMatrix2DViewTest(arg0: String) extends SparseCCMLongMatrix2DTest(arg0) {

  protected def createMatrices() {
    A = new SparseCCMLongMatrix2D(NCOLUMNS, NROWS).viewDice()
    B = new SparseCCMLongMatrix2D(NCOLUMNS, NROWS).viewDice()
    Bt = new SparseCCMLongMatrix2D(NROWS, NCOLUMNS).viewDice()
  }
}

File: src/test/java/cernx/colt/matrix/tlong/impl/SparseCCMLongMatrix2DViewTest.java
package cernx.colt.matrix.tlong.impl;

public class SparseCCMLongMatrix2DViewTest extends SparseCCMLongMatrix2DTest {

    public SparseCCMLongMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new SparseCCMLongMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new SparseCCMLongMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new SparseCCMLongMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }

}
